test/Frontend/test_datalayout.ll

```diff
@@ -0,0 +1,47 @@
+; RUN: %swiftc_driver_plain -frontend -target x86_64-apple-darwin10 -emit-object %s
```

Comment:
Is there a specific reason to use `%swiftc_driver_plain -frontend` instead of `%swift`?

Reply:
https://github.com/apple/swift/pull/16885